batch-submitter/drivers/sequencer/encoding_test.go

```diff
@@ -252,3 +255,27 @@ func TestIsMarkerContext(t *testing.T) {
 	}
 	require.True(t, batchContext.IsMarkerContext())
 }
+
+func TestReadUint64(t *testing.T) {
+	readUint64 := func(r io.Reader, val *uint64, n uint) error {
```

Comment:
Why do you define `readUint64` again here? Can we just call the `readUint64` in `batch-submitter/drivers/sequencer/encoding.go`?

Reply:
test file is in a different package, can't reference an unexported method